lms/static/scripts/frontend_apps/components/URLPicker.js

```diff
@@ -84,14 +82,13 @@ export default function URLPicker({ onCancel, onSelectURL }) {
         {/** setting a height here "preserves space" for this error display
          * and prevents the dialog size from jumping when an error is rendered */}
         <div
-          className="hyp-u-layout-row--center hyp-u-horizontal-spacing hyp-u-color--error"
+          className="h-4 flex flex-row items-center space-x-1 text-error"
```

Comment:
`text-error` from 'Tailwind utility classes based on the tailwind configuration', correct?

Reply:
`error` is another _color_ name in the configuration. I'm going to meditate a little on renaming these colors as I want this to be more obvious.

Reply:
Personally, I don't find the naming confusing. I was just curious about where this class was coming from.